src/Microsoft.OData.Core/ODataInnerError.cs

```diff
@@ -72,5 +75,24 @@ public ODataInnerError InnerError
             get;
             set;
         }
+
+        /// <summary>
+        /// Serilization to Json format string.
+        /// </summary>
+        /// <returns>The string in Json format</returns>
+        public override string ToString()

```

Comment:
yes, same as above.

Reply:
renamed as SerilizeToJsonString()

---
In reply to: [134865975](https://github.com/OData/odata.net/pull/951#discussion_r134865975) [](ancestors = 134865975)